src/python/WMCore/WorkQueue/WorkQueue.py

```diff
@@ -412,6 +412,16 @@ def _getDBSBlock(self, match, wmspec):
                                            acdcInfo['fileset'],
                                            splitedBlockName['Offset'],
                                            splitedBlockName['NumOfFiles'])
+            ### FIXME temporary debugging  ###
+            self.logger.info("AMR processing %d non-unique files for fileset %s",
+                             len(fileLists), acdcInfo['fileset'])
+            lumisPerRun = {}
+            for entry in fileLists:
+                for run in entry['runs']:
+                    lumisPerRun.setdefault(run.run, 0)
+                    lumisPerRun[run.run] += len(run.lumis)
+            self.logger.info("AMR run and # of lumis to be recovered: %s", lumisPerRun)
```

Comment:
do we need information for all the run lumis not by file?

Reply:
It should be enough to know the run and amount of lumis recovered for each of them.

Reply:
OK